packages/superset-ui-core/src/style/index.ts

```diff
@@ -20,7 +20,8 @@ import emotionStyled, { CreateStyled } from '@emotion/styled';
 import { useTheme as useThemeBasic } from 'emotion-theming';
 
 export { ThemeProvider, withTheme } from 'emotion-theming';
-export { css, InterpolationWithTheme } from '@emotion/core';
+export { css, InterpolationWithTheme, CacheProvider } from '@emotion/core';
```

Comment:
Let's also add `ClassNames` and `Global` as I see those are also used in superset

Reply:
Done.